### PR TITLE
copleted validations

### PR DESCRIPTION
**Pull request by XenaSit**

### CODE REVIEW

File: spec/models/tribute_spec.rb [hunk 1]
```diff
@@ -1,5 +1,53 @@
...
+    tribute = Tribute.create(
+      name: 'Katniss Everdeen',
+      age: nil, 
+      hobby: 'shooting up birds and squirrels', 
+      image: 'https://live.staticflickr.com/606/21731063948_c2bdd0aef6_b.jpg', 
+      district: 'District 12'
+    )
+    expect(tribute.errors[:age]).to_not be_empty
+  end
+  it 'should return length of hobby' do
```
Comment (SunkissedQueen) [inline spec/models/tribute_spec.rb:23]: Did you mean 'validate'?

File: spec/models/tribute_spec.rb [hunk 2]
```diff
@@ -1,5 +1,53 @@
...
+    tribute = Tribute.create(
+      name: 'Katniss Everdeen',
+      age: 16, 
+      hobby: 'shooting', 
+      image: 'https://live.staticflickr.com/606/21731063948_c2bdd0aef6_b.jpg', 
+      district: 'District 12'
+    )
+    expect(tribute.errors[:hobby]).to_not be_empty
+  end
+  it 'should return image' do
```
Comment (SunkissedQueen) [inline spec/models/tribute_spec.rb:33]: Did you mean 'validate'?

File: spec/models/tribute_spec.rb [hunk 3]
```diff
@@ -1,5 +1,53 @@
...
+    tribute = Tribute.create(
+      name: 'Katniss Everdeen',
+      age: 16, 
+      hobby: 'shooting', 
+      image: nil, 
+      district: 'District 12'
+    )
+    expect(tribute.errors[:image]).to_not be_empty
+  end
+  it 'should return district' do
```
Comment (SunkissedQueen) [inline spec/models/tribute_spec.rb:43]: Did you mean 'validate'?

File: spec/models/tribute_spec.rb (hunk 1 above)
Comment (XenaSit) [reply spec/models/tribute_spec.rb:23]: yes

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Hey team. During lectures, I might not be able to complete all the tasks. However, the goal is for your team to perform all the MVP tasks as outlined on the Trello.
Please complete the Requests checklist on the Trello card. Ping me on the same thread for your PR when the changes have been made.

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**